ceno_zkvm/src/bin/e2e.rs

```diff
@@ -77,22 +84,36 @@ fn main() {
     type Pcs = Basefold<GoldilocksExt2, BasefoldRSParams>;
     type ExampleProgramTableCircuit<E> = ProgramTableCircuit<E>;
 
-    // set up logger
-    let (flame_layer, _guard) = FlameLayer::with_file("./tracing.folded").unwrap();
```

Comment:
It looks like you are removing the flame related stuff here.  Do we still use it elsewhere?

(If not, we can probably remove the relevant crates from the dependencies in the `Cargo.toml`?)

You might also want to augment the PR description to mention that you are removing the flame related stuff?

Reply:
I reverted the change regarding the flamegraph. I think it's not too useful right now, but it is outside the scope of this PR.